tests/pyccel/scripts/kernel/hello_kernel.py

```diff
@@ -11,7 +11,7 @@ def say_hello(its_morning : bool):
 
 def f():
     its_morning = True
-    say_hello[1,1](its_morning)
+    say_hello[5,5](its_morning)
```

Comment:
Please don't change this.
It is not impossible that the output comes out as:
`Hello and Hello and Good morningGood morning`
or something horrible like this. Having a 1,1 test makes this impossible

Reply:
https://github.com/pyccel/pyccel-cuda/pull/69/commits/ef09298c9793b6fc1dda82edb0b53a07c97b3fd3